src/js/components/Table/Paginated/PaginatedTable.stories.tsx

```diff
@@ -0,0 +1,125 @@
+    position: "Goalie",
+    activePrograms: [
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" },
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" },
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" },
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" }
+    ]
+  },
+  {
+    name: "Dustin",
+    gender: "m",
+    age: 56,
+    position: "Coach",
+    activePrograms: [
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" },
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" }
+    ]
+  },
+  {
+    name: "Fred",
+    gender: "m",
+    age: 56,
+    position: "Coach",
+    activePrograms: [
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" },
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" }
+    ]
+  },
+  {
+    name: "Bobby",
+    gender: "m",
+    age: 56,
+    position: "Coach",
+    activePrograms: [
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" },
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" }
+    ]
+  },
+  {
+    name: "Christie",
+    gender: "f",
+    age: 56,
+    position: "Coach",
+    activePrograms: [
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" },
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" }
+    ]
+  },
+  {
+    name: "Jenna",
+    gender: "f",
+    age: 56,
+    position: "Coach",
+    activePrograms: [
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" },
+      { name: "2019 ACC Academy", subtitle: "(Junior Academy Tryout)" }
+    ]
+  }
+]);
+
+/**
+ * This function is where all of your server calls should occur. Commonly likely just make a
+ * server call and return the list of items. However, this is also a good place where you can
+ * inspect the request and determine how many items the server has for this search.
+ * @param page  - the table's current page
+ * @param itemsPerPage - the table's current number of items per page
+ * @param sortBy - the key that the table is sorting by
+ * @param isAscending boolean - true if ascending, false if not.
+ */
+function loadData(page, itemsPerPage, sortBy, isAscending) {
+  return data.then(items => [items[page - 1]]);
+}
+
+/**
+ * this function is defined to map a single data item into a single row item.
+ * It is passed in to the paginated table below.
+ * @param item the data item to be mapped to row data
+ */
+function mapData(item) {
+  return {
+    id: item.name,
+    name: (
+      <div>
+        <div>{item.name}</div>
+        <div>{item.position}</div>
+      </div>
+    ),
+    gender: item.gender,
+    age: `${item.age}`,
+    programs: item.activePrograms.map((p, idx) => <div key={idx}>{p.name}</div>)
+  };
+}
+
+stories.add("Default", () => (
+  <PaginatedTable
+    columns={columns}
+    mapDataToRow={mapData}
+    loadData={loadData}
+    defaultItemsPerPage={1}
```

Comment:
If you can set this to anything, how do we get the select list to match?

If you pass in a value that isn't one of the options, don't display the select?